peps/pep-9999.rst

```diff
@@ -14,41 +15,38 @@ Post-History: 02-Jun-2019,
-:external+py3.14:func:`math.gcd` or :external+py3.14:func:`math.isqrt`.
+This PEP proposes a new module for mathematical functions related to number
+theory and combinatorics.
 
 
 Motivation
 ==========
 
-The :external+py3.14:mod:`math` documentaton says: "This module provides access
-to the mathematical functions defined by the C standard."  But as a state of
-art, over time the module was populated with functions that aren't related to
-the C standard or floating-point arithmetics.  Now it's much harder to describe
-module scope, content and interfaces (returned values or accepted arguments).
-
-For example, the :external+py3.14:mod:`math` module documentation says: "Except
-when explicitly noted otherwise, all return values are floats."  This is not
-longer true:  *None* of the functions listed in the "Number-theoretic
-functions" [1]_ subsection of the documentation returns a float, but the
-documentation doesn't say so.  In the proposed module a similar sentence "All
-return values are integers." could tell the truth once.  In a similar way we
-can simplify description of accepted arguments for both the
-:external+py3.14:mod:`math` and the new module.
-
-Apparently, the :external+py3.14:mod:`math` can't serve as a catch-all place
-for mathematical functions: we have also the :external+py3.14:mod:`cmath` and
-the :external+py3.14:mod:`statistics`.  Let's make same for integer-related
-functions.  It would provide shared context, which reduces verbosity in the
-documentation and conceptual load.  It also aids discoverability through
-grouping related functions and IDEs suggesting helpful completions.
+The :external+py3.14:mod:`math` module documentation says: "This module provides
+access to the mathematical functions defined by the C standard."  But, over time
+the module was populated with functions that aren't related to those
+mathematical functions defined in the C standard.
+
+Also, the :external+py3.14:mod:`math` module documentation says: "Except when
+explicitly noted otherwise, all return values are floats."  This is no longer
+true:  *None* of the functions listed in the "Number-theoretic functions" [1]_
+subsection of the documentation returns a float, but the documentation doesn't
+say so. In the proposed ``imath`` module, where they belong, a similar
+catch-all: "Except when explicitly noted otherwise, all return values are
+integers." could tell the truth once.
+
+The ``imath`` module would parallel the :external+py3.14:mod:`math` module's
+other siblings: :external+py3.14:mod:`cmath` and the
+:external+py3.14:mod:`statistics`.  It would provide shared context, which
+reduces verbosity in the documentation and conceptual load.  It also aids
+discoverability through grouping related functions and IDEs suggesting helpful
+completions.
 
 
 Specification
 =============
 
-The PEP proposes moving the following integer-related functions [1]_ in a new
+The PEP proposes exposing the following integer-related functions [1]_ in a new
```

Comment:
No, "moving" is a correct word.  We move functions, period.

Reply:
We're not moving the functions though.  We're leaving them in `math` and _additionally exposing_ them in `imath`.

Are you talking about the implementations?